tensorboard/components_polymer3/experimental/plugin_util/message.ts

```diff
@@ -31,110 +31,97 @@ limitations under the License.
  * [1]: Using string to access property prevents JSCompiler mangling and make the
  * property stable across different versions of a bundle.
  */
-namespace tb_plugin.lib.DO_NOT_USE_INTERNAL {
-  export type PayloadType =
-    | null
-    | undefined
-    | string
-    | string[]
-    | boolean
-    | boolean[]
-    | number
-    | number[]
-    | object
-    | object[];
-
-  export interface Message {
-    type: string;
-    id: number;
-    payload: PayloadType;
-    error: string | null;
-    isReply: boolean;
+export type PayloadType =
```

Comment:
can we actually deduplicate with plugin_lib/message.ts instead? We can do that in a follow up if you'd like that.

Reply:
Yes!  Done in this PR.